docs/blog/2021-07-30-0.4.0-release.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Recoil 0.4
+---
+
+We are pleased to announce the release of Recoil 0.4 with initial Recoil Transactions, selector cache configuration, and other optimizations and fixes.
```

Comment:
"initial Recoil Transactions" doesn't make sense without further context. I would say "an improved API for updating multiple atoms together". So:

We are pleased to announce Recoil 0.4, with configurable selector caches, an improved API for updating multiple atoms together, and other improvements and fixes.